.gitmodules

```diff
@@ -16,3 +16,6 @@
 [submodule "lib/mento-core-2.3.1"]
 	path = lib/mento-core-2.3.1
 	url = https://github.com/mento-protocol/mento-core
+[submodule "lib/mento-core-2.4.0"]
```

Comment:
why do we need the new submodule?
I thought we had a submodule pointing to develop. On which we can just pull the latest on? (2.3.1 maybe)

Reply:
We should keep submodules pointing to the version if we need a `develop` one it should be temporarily added as `mento-core-develop`, once we have a version tagged it should be pinned. And I think Jamal cut a version for `2.4.0`